bcda/api/requests.go

```diff
@@ -470,13 +470,17 @@ func (h *Handler) bulkRequest(w http.ResponseWriter, r *http.Request, reqType se
 		err error
 	)
 
-	if ad, err = readAuthData(r); err != nil {
+	if ad, err = GetCtxAuthData(r); err != nil {
 		logger.Error(err)
 		h.RespWriter.Exception(r.Context(), w, http.StatusUnauthorized, responseutils.TokenErr, "")
 		return
 	}
 
-	rp, ok := middleware.RequestParametersFromContext(ctx)
+	if cfg, ok := h.Svc.GetACOConfigForID(ad.CMSID); ok {
+		ctx = service.NewCtxACOCfg(ctx, cfg)
```

Comment:
`Set`?

Reply:
Not sure I follow - do you mean this is where we are setting the ACO config in the context?